srcpkgs/frei0r-plugins/template

```diff
@@ -1,12 +1,16 @@
 # Template file for 'frei0r-plugins'
 pkgname=frei0r-plugins
-version=1.6.1
+version=1.7.0
 revision=1
 build_style=cmake
 hostmakedepends="autoconf"
+makedepends="$(vopt_if gavl gavl-devel) $(vopt_if cairo cairo-devel)"
 short_desc="Minimalistic API for a collection of free video effects plugins"
 maintainer="Diogo Leal <diogo@diogoleal.com>"
-license="GPL-2"
+license="GPL-2.0-or-later"
 homepage="http://frei0r.dyne.org/"
 distfiles="https://files.dyne.org/frei0r/releases/frei0r-plugins-${version}.tar.gz"
-checksum=e0c24630961195d9bd65aa8d43732469e8248e8918faa942cfb881769d11515e
+checksum=1b1ff8f0f9bc23eed724e94e9a7c1d8f0244bfe33424bb4fe68e6460c088523a
+
+build_options="cairo gavl"
+build_options_default="cairo"
```

Comment:
Why not enable both?

Also, their build system seems to search for OpenCV as well. Do we want to enable that?

Reply:
I thought to keep dependencies by default rather minimal and cairo is probably installed anyway, gavl is not.
I tried OpenCV (3 and 4) as well but it failed with different error messages. Maybe someone else knows how to do this...

Reply:
Fair enough, this should be adding more functionality as is already.

Can you leave a comment about OpenCV also being a possible dependency? `# XXX: ...`

Reply:
sry what do you mean by that `# XXX: ...` what comes instead of `XXX` ? :smile:

Reply:
ohh I didn't know it was a convention! nvm